wellcomecollection/aws-account-infrastructure
# Adds a GitHub action IAM role for the storage service repo

## What does this change?

Adds a new IAM role, allowing a GitHub action in the storage-service repository to deploy the Ingest Inspector app.

## How to test

See https://github.com/wellcomecollection/storage-service/pull/1133. If the GitHub action doesn't fail, the permissions were set up correctly. 

## How can we measure success?

The `deploy_ingest_inspector` GitHub action successfully deploys the Ingest Inspector backend and frontend whenever we merge to main.

## Have we considered potential risks?

The new role policy follows least privilege access principles, which minimises the risk of a buggy GitHub action removing or modifying critical infrastructure.
